monitoring/uss_qualifier/configurations/dev/faa/uft/local_message_signing.yaml

```diff
@@ -9,8 +9,17 @@ resources:
         flight_planners:
         - participant_id: uss1
           injection_base_url: http://host.docker.internal:8074/scdsc
+        - participant_id: uss2
```

Comment:
If we want mock_uss to be the second uss in flight planner combination, then would we need uss2?

Reply:
The way this PR is currently set up, all the F3548-21 tests will be run while mock_uss is logging, but only USS combinations 1) including mock_uss and 2) not having mock_uss in more than one role will be used.  With uss1, uss2, and mock_uss all listed as flight planners here, that means we'll see the following combinations for the nominal planning scenario:

| First-mover USS | Blocked USS |
|---|---|
| uss1 | mock_uss |
| uss2 | mock_uss |
| mock_uss | uss1 |
| mock_uss | uss2 |

Basically, uss1 and uss2 will both be tested with mock_uss present.  Putting uss1 and uss2 just in the First-mover USS role would be incomplete as they would make no outgoing calls to mock_uss.

Reply:
Sound good.
As we ll have only on uss under test at a time for message signing. 
So, we will probably go for a combination like -
uss1 | mock_uss 
mock_uss | uss1.

Hope that will be ok?

Reply:
Yes -- to test just one USS at a time, only uss1 and mock_uss would be listed as flight planners in the configuration.